test/test_core.py

```diff
@@ -760,42 +760,26 @@ def test_zero_multiplication(self):
   def test_isnan(self):
     self.do_core_test('test_isnan.c')
 
+  @only_wasm2js('tests globals in static data')
   def test_globaldoubles(self):
     self.do_core_test('test_globaldoubles.c')
 
   def test_math(self):
     self.do_core_test('test_math.c')
 
-  def test_erf(self):
-    self.do_core_test('test_erf.c')
-
-  def test_math_hyperbolic(self):
-    self.do_core_test('test_math_hyperbolic.c')
-
+  @only_wasm2js('tests lgamma and signbit')
   def test_math_lgamma(self):
     self.do_run_in_out_file_test('math/lgamma.c', assert_returncode=NON_ZERO)
 
+  @only_wasm2js('tests fmodf (which may use JS math)')
   def test_math_fmodf(self):
     self.do_run_in_out_file_test('math/fmodf.c')
```

Comment:
Should these tests also be moved in other and run in wasm2js mode?  Or do they benefit from being runing at different opt levels of wasm2js?

Reply:
I think these benefit from multiple opt levels, yeah. If we lower fmodf into JS `%` then the optimizer might do things with that. (However I don't think we actually do so, but it would be nice to someday.)